nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenExpr.scala

```diff
@@ -1517,11 +1534,11 @@ trait NirGenExpr { self: NirGenPhase =>
       val Seq(lengthp) = argsp
       val length = genExpr(lengthp)
 
-      buf.arrayalloc(genType(targ), length, unwind)
+      buf.arrayalloc(genType(targ), length, unwind)(targ.sym.pos)
     }
 
     def genApplyNew(clssym: Symbol, ctorsym: Symbol, args: List[Tree]): Val = {
-      val alloc = buf.classalloc(genTypeName(clssym), unwind)
+      val alloc = buf.classalloc(genTypeName(clssym), unwind)(clssym.pos)
```

Comment:
This is another instance of the above problem. I guess there are others scattered here and there.